Gems/ROS2/Code/Source/Frame/ROS2FrameEditorComponent.cpp

```diff
@@ -31,21 +35,26 @@ namespace ROS2
 
     void ROS2FrameEditorComponent::Init()
     {
-        m_configuration.m_namespaceConfiguration.Init();
     }
 
     void ROS2FrameEditorComponent::Activate()
     {
+        AZ_Printf("BuildGameEntity", "ROS2FrameEditorComponent::Activate for entity %s", this->GetEntityId().ToString().c_str());
```

Comment:
maybe  AZ_Trace to reduce number of logs?
```suggestion
        AZ_Trace("BuildGameEntity", "ROS2FrameEditorComponent::Activate for entity %s", this->GetEntityId().ToString().c_str());
```